win/rl/pynethack.cc

```diff
@@ -71,15 +72,21 @@ class Nethack
         }
     }
 
-    Nethack(std::string dlpath, std::string hackdir, bool spawn_monsters)
+    Nethack(std::string dlpath, std::string hackdir,
+            std::string nethackoptions, bool spawn_monsters)
         : dlpath_(std::move(dlpath)), obs_{}, spawn_monsters_(spawn_monsters)
     {
         if (hackdir.size() > sizeof(settings_.hackdir) - 1) {
             throw std::length_error("hackdir too long");
         }
+        if (nethackoptions.size() > sizeof(settings_.options)) {
```

Comment:
is one of these off by one?

Reply:
Good question. The reason for the difference is that in [nle.c](https://github.com/facebookresearch/nle/blob/heiner/nledl-interface/src/nle.c#L173) we need to add a `/` at the end. This is perhaps a bit unclean, but reflects what NetHack itself does in a comparable situation.

Reply:
But that `/` only needs to be added to HACKDIR.